news/2020-03-25-new-featured-artist-o2i3.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: O2i3"
+date: 2020-03-25 08:00:00 +0000
+---
+
+We're proud to welcome **O2i3** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/74/header.jpg)
+
+Jumping into our ears back during 2015 with *Ooi*, **O2i3** is a Korean producer with a penchant for energetic and engaging tracks drawing from a variety of electronic music genres for inspiration.
+
+We've managed to snag **8** tracks of his, including classics like *TSLove* and a few others that we'll think you'll love.
+
+Check it all out:
+
+### Ooi
+
+Check out [this hugely popular set](https://osu.ppy.sh/beatmapsets/355573) by [Ciyus Miapah](https://osu.ppy.sh/users/2805457)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/74/previews/2011.mp3" type="audio/mpeg">
+</audio>
+
+### TSLove
+
+Check out this [popular set](https://osu.ppy.sh/beatmapsets/620910) hosted by [Fanteer](https://osu.ppy.sh/users/7989641)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/74/previews/2013.mp3" type="audio/mpeg">
+</audio>
+
+### Heart Function
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/74/previews/2007.mp3" type="audio/mpeg">
+</audio>
+
+### Melco, Melco
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/74/previews/2010.mp3" type="audio/mpeg">
+</audio>
+
+### Dollfie Dream
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/74/previews/2006.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+All of the above plus a few more are available right now over at [**O2i3**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/74), all pre-timed and awaiting your creative touch. Maybe you're the next Fort in the making, give it a try!
+
+The hint tweet arrived fashionably late this week, but that didn't stop tweeters @Kaeldori, @Daykeyboard, @Qauland and @idealismosu (plus others who arrived similarly late) from guessing correctly. Good job!
+
+I think everybody all knows by now what will happen for next week's release, but in case you don't, check in with the [@osugame Twitter](https://twitter.com/osugame) throughout for hints, tidbits and other stuff you'll probably like. See you then!
```

Comment:
everybody all knows? bad english or some kind of hint?

Reply:
Let's blame Fort..

Reply:
Plus, someone already noticed a O2i3 ranked taiko map just before annoucement. Massive red flag